pcsx2/Counters.h

```diff
@@ -93,8 +93,8 @@ struct SyncCounter
 //------------------------------------------------------------------
 #define FRAMERATE_NTSC			29.97 // frames per second
 
-#define SCANLINES_TOTAL_NTSC	525 // total number of scanlines
-#define SCANLINES_VSYNC_NTSC	3   // scanlines that are used for syncing every half-frame
+#define SCANLINES_TOTAL_NTSC	525   // total number of scanlines
+#define SCANLINES_VSYNC_NTSC	3    // scanlines that are used for syncing every half-frame
 #define SCANLINES_VRENDER_NTSC	240 // scanlines in a half-frame (because of interlacing)
```

Comment:
The spacing on the comments could be adjusted. -2 on the first row, -1 on the second.

Reply:
@lightningterror this is due to GitHub